charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,182 @@
+# Default values for fluentbit-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Set this to containerd or crio if you want to collect CRI format logs
+containerRuntime: docker
+# You can also deploy the Kubernetes logging stack with Helm, just need to set the Kubernetes parameter to ture.
+# see https://github.com/fluent/fluent-operator/tree/master/manifests/logging-stack
+Kubernetes: false
+
+operator:
+# The initial container is to get the actual storage path of the docker log files so that it can be mounted to collect the logs.
+# see https://github.com/fluent/fluent-operator/blob/master/manifests/setup/fluent-operator-deployment.yaml#L26
+  initcontainer:
+    repository: "docker"
+    tag: "20.10"
+  container:
+    repository: "kubesphere/fluent-operator"
+    tag: "v1.7.0"
+    # FluentBit operator resources. Usually user needn't to adjust these.
+  resources:
+    limits:
+      cpu: 100m
+      memory: 60Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  # For non-docker environments, the mount path for fluentbit uses a hard-coded path.
+  logPath:
+    crio: /var/log
+    containerd: /var/log
+
+fluentbit:
+  image:
+    repository: "kubesphere/fluent-bit"
+    tag: "v1.9.9"
+  # fluentbit resources. If you do want to specify resources, adjust them as necessary
+  # You can adjust it based on the log volume.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 200Mi
+    requests:
+      cpu: 10m
+      memory: 25Mi
+  # Specify custom annotations to be added to each FluentBit pod.
+  annotations: {}
+    # prometheus.io/scrape: "true"
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  secrets: []
+  # List of volumes that can be mounted by containers belonging to the pod.
+  volumes: []
+  # Pod volumes to mount into the container's filesystem.
+  volumesMounts: []
+
+  # Remove the above empty volumes and volumesMounts, and then set volumes and volumesMounts as below if you want to collect node exporter metrics
```

Comment:
Similar to the above question - this would not affect log tailing if you did this? Can we state that if so. 

What if you don't want to mount any logs into the container, e.g. just to do metrics/OTEL? I know it's unlikely but curious if it is possible.

Reply:
This will not affect log tailing and the below section is to collect node-exporter metrics only and has nothing to do with logs